bin/bump_version.py

```diff
@@ -121,18 +121,9 @@ def bump_version():
     while input('Type "done" to continue: ').strip().lower() != 'done':
         pass
 
-    subprocess.run([
-        'git', 'commit',
-        '-a',
-        '-m', f'Bump version: v{new_version}'
-    ], check=True)
-
-    subprocess.run([
-        'git', 'tag',
-        '-a',
-        '-m', f'v{new_version}',
-        f'v{new_version}'
-    ], check=True)
+    subprocess.run(['git', 'commit', '-a', '-m', f'Bump version: v{new_version}'], check=True)
```

Comment:
There seems to be a +- consistent style (I guess by @joerick?) where flags of a subprocess call are always on new lines. So this actually introduces inconsistency one some level.

Reply:
If we did want to go with black, we could add fmt: on / off around these. It's okay to have special style for certain things and consistent style everywhere else. Or add a trailing comma, which would do one per line.

Reply:
> Or add a trailing comma, which would do one per line.

The thing is that things that go together (`'-m'` and `f'v{new_version}'`) are together here, though.

Reply:
Then use `f"--message=v{new_version}"`. All together now, and even clearly says what is being done instead of "m". :)

Reply:
This is generally the solution, by the way - you should not use single char flags in a script, you should write it out - there's no need to save characters in a script. And if you do, it's then in one line instead of two using the `=` syntax.